app/assets/stylesheets/welcome.scss

```diff
@@ -160,6 +160,29 @@ span.list-group-item.active {
   color: #e573fc;
   font-size: 10px;
 }
+
 .url-list {
   padding-bottom: 200px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #ff6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
+          background-color: #a4e7a0;
+        }
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.